pkg/util/kubernetes/apiserver/openshift_types.go

```diff
@@ -0,0 +1,18 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+// +build kubeapiserver
+
+package apiserver
+
+// OpenShiftAPILevel describes what level of OpenShift APIs are available on the apiserver
+type OpenShiftAPILevel string
```

Comment:
Does `OpenShift` have any other apigroups/custom resources other than clusterquotas we may want to collect metrics for in the future?

Should this maybe be called `OpenShiftQuotasAPILevel`

```
OpenShiftQuotasAPIGroup
OpenShiftQuotasOAPI
OpenShiftQuotasNone
```